org.eclipse.lsp4e.test/src/org/eclipse/lsp4e/test/linkedediting/LinkedEditingTest.java

```diff
@@ -141,13 +141,12 @@ protected boolean condition() {
 		}.waitForCondition(Display.getCurrent(), 3000);
 
 		IAnnotationModel model = sourceViewer.getAnnotationModel();
-		Annotation annotation = findAnnotation(sourceViewer, 14);
-		Assert.assertNotNull(annotation);
-//		System.out.println("testLinkedEditingExitPolicy: annotation found at offset 14: " + annotation.getType());
-		assertTrue(annotation.getType().startsWith("org.eclipse.ui.internal.workbench.texteditor.link"));
+		List<Annotation> annotations = findAnnotations(sourceViewer, 14);
+		Assert.assertFalse(annotations.isEmpty());
```

Comment:
We don't need to check that.

Reply:
Hi @mickaelistria you've already cherry-picked the commit and fixed this, is that right?